package.json

```diff
@@ -4,38 +4,35 @@
   "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
-
-  "test": "nyc --reporter=html --reporter=text mocha **/*_test.js -r @babel/polyfill -r @babel/register --exit",
+    "test": "npm run migration:refresh:test && npm run db:seed:test && NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout 20000 -r @babel/register ./src/modules/**/*.test.js --exit",
     "start": "node dist",
     "build": "rm -rf dist/ && babel src/ -d dist/",
-     "start:dev": "nodemon --exec babel-node ./src/index.js",
-    "sequelize": "sequelize db:migrate all",
+    "start:dev": "npm run migration:refresh && npm run db:seed && nodemon --exec babel-node ./src/index.js",
+    "migration": "sequelize db:migrate",
+    "migration:rollback": "sequelize db:migrate:undo:all",
+    "migration:refresh": "npm run migration:rollback && npm run migration",
+    "db:seed": "sequelize db:seed:all",
+    "migration:test": "NODE_ENV=test sequelize db:migrate",
+    "migration:rollback:test": "NODE_ENV=test sequelize db:migrate:undo:all",
+    "migration:refresh:test": "NODE_ENV=test npm run migration:rollback:test && npm run migration:test",
+    "db:seed:test": "NODE_ENV=test sequelize db:seed:all",
     "lint": "./node_modules/.bin/eslint src",
     "lint-fix": "./node_modules/.bin/eslint src --ext  .js --fix",
-    "heroku-postbuild": "npm run build"
-},
+    "heroku-postbuild": "npm run build",
```

Comment:
modify your build script to look to include database setup and seeding
`"heroku-postbuild": "npm run migration && npm run db:seed && npm run build"`

Reply:
Please let's leave the `start:dev` script the way it is. Migrations and seeders can be run individually when there's the need to. Having the migration and seed command running anytime I start my local server can become annoying.

Reply:
okay